src/pages/FriendsPage.tsx

```diff
@@ -0,0 +1,221 @@
+  const me = useRecoilValue<userInterface>(meAtom);
+  const friendsList = useRecoilValue<userInterface[]>(friendsListAtom);
+  const [isSearching, setIsSearching] = useState<boolean>(false);
+  const [isFriendsOpen, setIsFriendsOpen] = useState<boolean>(true);
+  const [isProfileModal, setIsProfileModal] = useState<boolean>(false);
+  const [clickedProfileInfo, setClickedProfileInfo] = useState<userInterface>({
+    userId: 0,
+    userName: "",
+    profileImage: "",
+    statusMessage: "",
+  });
+
+  const handleRightClick = (e: React.MouseEvent<HTMLDivElement>) => {
+    e.preventDefault();
+  };
+
+  const handleProfileClick = () => {
+    setIsProfileModal(true);
+  };
+
+  return (
+    <PageWrapStyled onContextMenu={handleRightClick}>
+      <Splash />
+
+      <Navigation menu={"friends"} />
+
+      <PageMainStyled>
+        <Header>
+          <HeaderTitle>
+            <span>친구</span>
+            <IconWrapper onClick={() => setIsSearching(!isSearching)}>
+              <Search width={18} height={18} />
+            </IconWrapper>
+          </HeaderTitle>
+
+          {isSearching && (
+            <HeaderInput>
+              <input
+                value={inputText}
+                onChange={handleInputChange}
+                placeholder="이름 검색"
+                autoFocus
+                spellCheck="false"
+              ></input>
+              <IconWrapper onClick={() => setIsSearching(false)}>
+                <Close width={12} height={12} />
+              </IconWrapper>
+            </HeaderInput>
+          )}
+        </Header>
+
+        <Body>
+          <SingleCard
+            type="profile"
+            userInfo={me}
+            setClickedProfileInfo={setClickedProfileInfo}
+            handleProfileClick={handleProfileClick}
+          />
+
+          <Hr></Hr>
+
+          <Dropdown>
+            <span>
+              친구{" "}
+              {
+                friendsList.filter((user) => user.userName.includes(inputText))
+                  .length
+              }
+            </span>
+            <IconWrapper onClick={() => setIsFriendsOpen(!isFriendsOpen)}>
+              {isFriendsOpen ? (
+                <DownArrow width={10} height={10} color={"var(--gray-font)"} />
+              ) : (
+                <UpArrow width={10} height={10} color={"var(--gray-font)"} />
+              )}
+            </IconWrapper>
+          </Dropdown>
+
+          <Hr></Hr>
+
+          {isFriendsOpen &&
+            friendsList.map((user) => {
+              if (user.userName.includes(inputText)) {
+                return (
+                  <SingleCard
+                    key={user.userId}
+                    type="profile"
+                    userInfo={user}
+                    setClickedProfileInfo={setClickedProfileInfo}
+                    handleProfileClick={handleProfileClick}
+                  />
+                );
+              }
+            })}
+        </Body>
```

Comment:
현재 대소문자까지 비교해서 search로 걸러주고 있는데 아래처럼 다 소문자로 바꿔서 비교하거나 다 대문자로 바꿔서 비교해주면 더 편리하게 검색할 수 있게 구현할 수 있을 것 같아요 ! 
```suggestion
          {isFriendsOpen &&
            friendsList.map((user) => {
              if (user.userName.toLowerCase().includes(inputText.toLowerCase())) {
                return (
                  <SingleCard
                    key={user.userId}
                    type="profile"
                    userInfo={user}
                    setClickedProfileInfo={setClickedProfileInfo}
                    handleProfileClick={handleProfileClick}
                  />
                );
              }
            })}
        </Body>
```